.changeset/short-starfishes-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/libs': patch
+---
+
+Updating the Australian Privacy Manager id
```

Comment:
Remember that this message is for anyone who uses `@guardian/libs`, so it's useful to be super clear about a change and the reasons for it i.e. "do i need to know about this or can i ignore this one?".

Not sure if I've got the gist of it here, but maybe something like this?

```suggestion
Internally, `cmp.init` uses a new id for Sourcepoint consent config in Australia.

Reflects a change to our setup at Sourcepoint.
```